package.json

```diff
@@ -47,11 +48,13 @@
     "babel-core": "^6.18.2",
     "babel-eslint": "^7.1.1",
     "babel-loader": "^6.2.4",
-    "babel-plugin-react-transform": "^2.0.2",
+    "babel-plugin-transform-class-properties": "^6.19.0",
+    "babel-plugin-transform-decorators-legacy": "^1.3.4",
```

Comment:
Yeah, I meant decorators. I think we are only using Object rest spread, class properties and maybe export extensions.